web/api/mastermind/order.js

```diff
@@ -0,0 +1,72 @@
+const Joi = require('joi');
+const auth = require('../../lib/authentications');
+const validation = require('../validations/orders');
+const orderHandlers = require('../../lib/handlers/order');
+
+const basePath = '/api/3.0';
+const { applications } = validation.definitions;
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${basePath}/events/{eventId}/orders/{orderId}`,
+    handler: orderHandlers.get(),
+    config: {
+      auth: auth.apiUser,
+      description: 'List A User\'s Order',
+      notes: 'User\'s Order',
+      tags: ['api', 'orders'],
+      plugins: {
```

Comment:
Missing permissions? 
We don't want to allow everybody to list everybody's orders (which we don't because it's based on the current user Id), yet, an o13 should be allowed to see he's already booked in by its parent

Reply:
I see. I have to make a new permission for this?

Reply:
Mmmh. We don't currently have a use case for listing orders.
So maybe let it be until then cc @DanielBrierton  ?

Reply:
We have a use case for getting a specific order – the confirmation page.

Reply:
Mmh, ok, but it won't be accessible to the children being booked. So it doesn't apply in this scenario, so it could be alright in the current state

Reply:
Well, we'll need to confirm that the user requesting it is the creator. But I think the handler is using the userId of the logged in user, so I guess it should be fine?

Reply:
Yup, nothing to see here, skip :)
Thanks !